lib/services/sku-pack-service.js

```diff
@@ -38,7 +44,10 @@ function skuPackServiceFactory(
     assert,
     workflowApiService,
     Constants,
-    Env
+    Env,
```

Comment:
The first letter should be in lower-case (`Env` ==> `env`), since `Services.Environment` returns an instance not a Class.